mxcube3/routes/qutils.py

```diff
@@ -1483,6 +1483,11 @@ def execute_entry_with_id(sid, tindex=None):
         node_id = current_queue[sid]["tasks"][int(tindex)]["queueID"]
 
         node, entry = get_entry(node_id)
+	# in order to fill lims data, we execute first the parent (group_id missing)
+	parent_id = node.get_parent()._node_id
```

Comment:
Watch out,  indent seems to be off by one ?

Reply:
good catch! thanks